test/hotspot/jtreg/containers/docker/TestMemoryAwareness.java

```diff
@@ -79,6 +87,9 @@ public static void main(String[] args) throws Exception {
                 "1G", Integer.toString(((int) Math.pow(2, 20)) * 1024),
                 "1500M", Integer.toString(((int) Math.pow(2, 20)) * (1500 - 1024))
             );
+            hostMaxMem = getHostMaxMemory();
+            testOperatingSystemMXBeanIgnoresMemLimitExceedingPhysicalMemory();
+            testMetricsIgnoresMemLimitExceedingPhysicalMemory();
             testContainerMemExceedsPhysical();
```

Comment:
Could we pass `hostMaxMem` to the relevant test functions as a parameter, instead of relying on a static variable magically be set, please? It would be a good symmetry to other tests.

Reply:
Good idea! Implemented in 0f37be83ff0.